bedrock/security/templates/security/bug-bounty/faq-webapp.html

```diff
@@ -193,8 +309,8 @@ <h1 itemprop="name" class="title-shadow-box">Web Application Security Bug Bounty
         <dt class="question" id="what-next">Once I have found a
           vulnerability, what next?</dt>
         <dd class="answer">
-          <p>Please <a href="https://bugzilla.mozilla.org/form.web.bounty">
-            file a bug</a>.
+          <strong><p>Follow the <a href="https://www.mozilla.org/en-US/security/web-bug-bounty/">guide on submitting a web or service security bug</a> and then <a href="https://bugzilla.mozilla.org/form.web.bounty">
```

Comment:
Also, should move the opening `<p>` tag before the opening `<strong>` for proper nesting.